tools/replace_code_block.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+from traceback import print_tb
+from typing import AnyStr
+
+status = 0
+
+
+def check_for_code(file: AnyStr, check: bool):
+    """Check a file and replace ``.. code::`` to ``.. code-block::``
+
+    :param file: The file to open
+    :param check: Whether or not the program is operating in ``check`` mode.
```

Comment:
Can we rename check to `overwrite` or something else to indicate it's the more destructive action?

Reply:
But `overwrite` is the default action.

Reply:
Right, overwrite should not be the default.